e2e/nomostest/nt.go

```diff
@@ -870,7 +870,7 @@ func (nt *NT) ForwardToFreePort(ns, pod, port string) (int, error) {
 		cleanup = true
 		nt.T.Log("stopping port-forward %s/%s:%s process", ns, pod, port)
 		err := cmd.Process.Kill()
-		if err != nil {
+		if err != nil && !errors.Is(err, os.ErrProcessDone) {
```

Comment:
I have code to change this to use Context cancel in https://github.com/GoogleContainerTools/kpt-config-sync/pull/412, but this works for now. Thanks!